frameworks/elastic/universe/config.json

```diff
@@ -34,6 +34,11 @@
           "description": "Elasticsearch deploy strategy. [parallel, serial]",
           "type": "string",
           "default": "parallel"
+        },
+        "mesos_api_version": {
```

Comment:
OK per conversation elsewhere this option will likely be removed from everything except hello-world soon. Can just add it here for now as a no-op for CI compatibility, and then remove it again in a later pass.